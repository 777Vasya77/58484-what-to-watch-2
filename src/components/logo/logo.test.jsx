import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '~/components/logo/logo';

jest.mock(`react-router-dom`, () => ({
  Link: `Link`
}));

describe(`Logo component tests`, () => {
  it(`Component render correctly`, () => {
    const tree = renderer
        .create(
            <Logo />
        ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
