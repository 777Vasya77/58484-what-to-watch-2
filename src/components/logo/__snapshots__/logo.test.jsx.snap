// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component tests Component render correctly 1`] = `
<div
  className="logo"
>
  <Link
    className="logo__link logo__link--light"
    style={
      Object {
        "borderColor": "unset",
        "color": "unset",
      }
    }
    to="/"
  >
    <span
      className="logo__letter logo__letter--1"
    >
      W
    </span>
    <span
      className="logo__letter logo__letter--2"
    >
      T
    </span>
    <span
      className="logo__letter logo__letter--3"
    >
      W
    </span>
  </Link>
</div>
`;
